packages/contracts/contracts/test/DepositFeed.t.sol

```diff
@@ -148,31 +147,11 @@ contract DepositFeedTest is DSTest {
         assertEq(address(df).balance, NON_ZERO_VALUE);
     }
 
-    // Test: depositTransaction should increase its eth balance when an EOA deposits a transaction with ETH
```

Comment:
Any reason to delete this test?

Reply:
Because I moved `receive()` to the Portal contract.